install-wings.sh

```diff
@@ -428,8 +361,22 @@ function install_mariadb {
 ##### OS SPECIFIC FUNCTIONS #####
 #################################
 
-function firewall_ufw {
-  apt update
```

Comment:
Shouldn't the `apt update` be retained?

Reply:
No, because It updates one time only at the start and as far as I know` ufw` comes after that

![image](https://user-images.githubusercontent.com/26341983/98170018-f4558c00-1ef5-11eb-8f67-abaf965251c2.png)